### PR TITLE
Purchase Revoke

### PR DESCRIPTION
**Pull request by raimonesteve**
#072950

### CODE REVIEW

**Review by albertca — COMMENTED** [review]
The only comments would be the same that in the sale_revoke module.

Also this one misses a test.

**Comment by raimonesteve** [comment]
upload review: cf27417b31f558e52917e577820ea0e167e026bf
missing commit scenario: 8811f4fe143d42f6dd034a59aa0231bcd4c893c0

File: message.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@ this repository contains the full copyright notices and license terms. -->
 <tryton>
     <data grouped="1">
         <record model="ir.message" id="msg_can_not_revoke">
-            <field name="text">You cannot revoke "%(record)s" because must first cancel, draft or done the moves or shipments:
+            <field name="text">You cannot revoke "%(record)s" because you there are moves that are not in cancel, draft or done state:
```
Comment (albertca) [inline message.xml]: Same as sale_revoke

Comment (raimonesteve) [reply message.xml]: 6caa041377733953d188c4aa2345701b55038f86

**Review by albertca — APPROVED** [review]

**Pull request merged**